src/components/grid-aware/VideoSpotlightBlock/VideoSpotlightBlock.module.css

```diff
@@ -0,0 +1,109 @@
+.bleedWrapper {
+  background-color: var(--color-black);
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  grid-template-rows: 465px;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+}
+
+.bleedMainContent {
+  grid-column: 2;
+  /* Override grid item default of auto so that item contents cannot stretch the grid itself */
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    display: flex;
+    flex-direction: column;
+  }
+
+  .bleedBackground {
+    height: 250px;
+  }
+}
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: block;
+  }
+}
+
+.textCardWrapper {
+  text-align: center;
+}
+
+@media (--laptop-and-up) {
+  .gridAreaImage {
+    align-items: center;
+    display: flex;
+    justify-content: center;
+    position: absolute;
+  }
```

Comment:
Couple notes on this:

1. As I requested in the JSX, I think this should really be named something more like `bleedImage`, and it should also be moved closer to the elements more related to it. This has more in common with `bleedMainContent` as well as `playButton` and `image` than it does with `gridAreaCard`.

2. I think your positioning and centering is a bit overly complicated here, since you have multiple layers of absolute position and weird stuff going on with flex and centering. Could you:
    1. Ditch the flexbox altogether; it's kind of confusing to reason about the position of the play button and the image with the flexbox and with the absolute positioning of the play button
    2. Also ditch the `align-items: center` and `justify-content: center`.
    3. Continue to position the `playButton` absolutely, but use the old-school way of centering an element vertically and horizontally through the use of `top: 50%`, `left: 50%`, and `transform: translate(-50%, -50%)`?

Reply:
okay